packages/notebook-extension/src/index.ts

```diff
@@ -609,21 +610,24 @@ export const exportPlugin: JupyterFrontEndPlugin<void> = {
           return;
         }
 
-        const url = PageConfig.getNBConvertURL({
-          format: args['format'] as string,
-          download: true,
-          path: current.context.path
-        });
         const { context } = current;
 
+        const exportOptions: NbConvert.IExportOptions = {
+          format: args['format'] as string,
+          path: current.context.path,
+          exporterOptions: {
+            download: true
+          }
+        };
+
         if (context.model.dirty && !context.model.readOnly) {
           return context.save().then(() => {
-            window.open(url, '_blank', 'noopener');
+            void services.nbconvert.exportAs?.(exportOptions);
           });
         }
 
         return new Promise<void>(resolve => {
-          window.open(url, '_blank', 'noopener');
+          void services.nbconvert.exportAs?.(exportOptions);
```

Comment:
What is the rationale for `void` over await here?

Reply:
I think it was just to keep the existing surrounding `return new Promise<void>`.

But maybe we could remove it now and simply return `services.nbconvert.exportAs?.(exportOptions)` ?